docs/repositories-configure/integrations/github-integration.md

```diff
@@ -43,6 +43,19 @@ Depending on the options that you enable, Codacy will automatically update pull
 
     ![Pull request summary on GitHub](images/github-integration-pr-summary.png)
 
-You can find all the information reported by Codacy on the tab **Checks** of your pull requests:
+-   **Suggest fixes** adds comments on the lines of the pull request where Codacy finds new issues with suggestions on how to fix the issues. Codacy doesn't make any changes automatically. To apply the changes, manually review and accept the suggestions.
+
+    <!-- TODO Add screenshot -->
+
+    !!! note
+        This feature is in early access and has the following limitations:
+
+        -   The only supported Git providers are GitHub Cloud and GitHub Enterprise
+        -   The only tool that suggests fixes is ESLint for JavaScript
+        -   The author of the comments is the user that enabled the GitHub integration for the Codacy repository
+
+        📢 [Let us know](mailto:support@codacy.com?subject=Feedback on Suggest fixes) what you think about this feature!
```

Comment:
@davidscordeiro, what do you think about openly presenting this feature as "early access" (or [open beta](https://en.wikipedia.org/wiki/Software_release_life_cycle#Beta)), and actively encouraging customers to send us their feedback?

Reply:
By chance, I just found out that GitHub launched the "suggested changes" feature as a "public beta":

https://github.blog/changelog/2018-10-16-suggested-changes/